extensions/wikia/TemplateDraft/TemplateConverter.class.php

```diff
@@ -160,6 +181,74 @@ private function prepareVariables( $templateVariables ) {
 		return $variables;
 	}
 
+	/**
+	 * Prepares content in special format.
+	 * It doesn't parse template variables to extract them later.
+	 *
+	 * @param $content
+	 * @return string
+	 */
+	private function prepareContent( $content ) {
+		global $wgUser;
+
+		$ParserPool = \ParserPool::get();
```

Comment:
The variable name is misleading, as you're not getting a `ParserPool` object, you're getting a `Parser` object. It also should use lowerCamelCase. I'd just rename it to `$parser`.